cmd/gangway/main.go

```diff
@@ -47,6 +47,18 @@ func httpLogger(fn http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+func rootPathHandler(fn http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// The "/" pattern matches everything, so we need to check
+		// that we're at the root here.
+		if cfg.HTTPPath == "" && r.URL.Path != "/" {
+			http.NotFound(w, r)
```

Comment:
Maybe this should just 301 redirect back to `/`?

Reply:
Sounds good.